File: lua/test/exists_test.lua
-- ProjectName SDK exists test

local sdk = require("playstation-store_sdk")

describe("PlaystationStoreSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)

File: rb/test/exists_test.rb
# PlaystationStore SDK exists test

require "minitest/autorun"
require_relative "../PlaystationStore_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = PlaystationStoreSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
